PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalCheckoutRequest.kt

```diff
@@ -101,6 +103,7 @@ class PayPalCheckoutRequest @JvmOverloads constructor(
     override var recurringBillingDetails: PayPalRecurringBillingDetails? = null,
     override var recurringBillingPlanType: PayPalRecurringBillingPlanType? = null,
     var amountBreakdown: AmountBreakdown? = null,
+    var shouldOfferCredit: Boolean = false,
```

Comment:
It looks like the `shouldOfferCredit` property exists in the `PayPalVaultRequest` class. Can we add this property to the base `PayPalRequest` class and override it in both `PayPalCheckoutRequest` and `PayPalVaultRequest` classes?

Reply:
Good idea. Addressed in latest commit.